spacy/tokens/doc.pyx

```diff
@@ -1692,11 +1690,13 @@ cdef class Doc:
                             # Check if token attribute
                             elif end is None:
                                 if attr not in data["underscore_token"]:
-                                    data["underscore_token"][attr] = {"token_start": start, "value": value}
+                                    data["underscore_token"][attr] = []
+                                data["underscore_token"][attr].append({"token_start": start, "value": value})
```

Comment:
Since these are character offsets, I would use `start` and `end` like the other character span entries in `Doc.to_json`.